src/components/List/AddCard.js

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  FormControl,
+  InputLabel,
+  Select,
+  TextField,
+  withStyles,
+} from "@material-ui/core";
+import MuiDialogContent from "@material-ui/core/DialogContent";
+import MuiDialogActions from "@material-ui/core/DialogActions";
+import { useCurrentUser } from "../../hooks/useCurrentUser";
+import { getAllUserData } from "../../firebase/getAllUserData";
+import { addListEntry } from "../../firebase/addListEntry";
+import Snackbar from "../Snackbar";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: "white",
+    maxWidth: 500,
+    width: "100%",
+    height: "100%",
+  },
+  buttons: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  dialogTitle: {
+    textAlign: "center",
+  },
+  textField: {
+    display: "flex",
+    paddingBottom: theme.spacing(2),
+  },
+  formControl: {
+    display: "flex",
+    paddingBottom: theme.spacing(2),
+  },
+}));
+
+export default function AddCard({ open, close, cardTitle, list, triggerUpdate }) {
+  const classes = useStyles();
+  const [selectedGroup, setSelectedGroup] = useState("");
+  const [selectedUser, setSelectedUser] = useState("");
+  const [selectedDate, setSelectedDate] = useState("");
+  const [title, setTitel] = useState("");
```

Comment:
```suggestion
  const [title, setTitle] = useState("");
```
Don't mix Deutsch und Englisch 😉